test/cmd/admin.sh

```diff
@@ -41,188 +42,192 @@ status:
-oadm policy reconcile-cluster-roles --confirm
-oc get clusterrole/cluster-status
+os::cmd::expect_success 'oadm policy who-can get pods'
+os::cmd::expect_success 'oadm policy who-can get pods -n default'
+os::cmd::expect_success 'oadm policy who-can get pods --all-namespaces'
+
+os::cmd::expect_success 'oadm policy add-role-to-group cluster-admin system:unauthenticated'
+os::cmd::expect_success 'oadm policy add-role-to-user cluster-admin system:no-user'
+os::cmd::expect_success 'oadm policy add-role-to-user admin -z fake-sa'
+os::cmd::expect_success_and_text 'oc get rolebinding/admins -o jsonpath={.subjects}' 'fake-sa'
+os::cmd::expect_success 'oadm policy remove-role-from-user admin -z fake-sa'
+os::cmd::expect_success_and_not_text 'oc get rolebinding/admins -o jsonpath={.subjects}' 'fake-sa'
+os::cmd::expect_success 'oadm policy add-role-to-user admin -z fake-sa'
+os::cmd::expect_success_and_text 'oc get rolebinding/admins -o jsonpath={.subjects}' 'fake-sa'
+os::cmd::expect_success "oadm policy remove-role-from-user admin system:serviceaccount:$(oc project -q):fake-sa"
+os::cmd::expect_success_and_not_text 'oc get rolebinding/admins -o jsonpath={.subjects}' 'fake-sa'
+os::cmd::expect_success 'oadm policy remove-role-from-group cluster-admin system:unauthenticated'
+os::cmd::expect_success 'oadm policy remove-role-from-user cluster-admin system:no-user'
+os::cmd::expect_success 'oadm policy remove-group system:unauthenticated'
+os::cmd::expect_success 'oadm policy remove-user system:no-user'
+os::cmd::expect_success 'oadm policy add-cluster-role-to-group cluster-admin system:unauthenticated'
+os::cmd::expect_success 'oadm policy remove-cluster-role-from-group cluster-admin system:unauthenticated'
+os::cmd::expect_success 'oadm policy add-cluster-role-to-user cluster-admin system:no-user'
+os::cmd::expect_success 'oadm policy remove-cluster-role-from-user cluster-admin system:no-user'
+
+os::cmd::expect_success 'oadm policy add-scc-to-user privileged fake-user'
+os::cmd::expect_success_and_text 'oc get scc/privileged -o yaml' 'fake-user'
+os::cmd::expect_success 'oadm policy add-scc-to-user privileged -z fake-sa'
+os::cmd::expect_success_and_text 'oc get scc/privileged -o yaml' "system:serviceaccount:$(oc project -q):fake-sa"
+os::cmd::expect_success 'oadm policy add-scc-to-group privileged fake-group'
+os::cmd::expect_success_and_text 'oc get scc/privileged -o yaml' 'fake-group'
+os::cmd::expect_success 'oadm policy remove-scc-from-user privileged fake-user'
+os::cmd::expect_success_and_not_text 'oc get scc/privileged -o yaml' 'fake-user'
+os::cmd::expect_success 'oadm policy remove-scc-from-user privileged -z fake-sa'
+os::cmd::expect_success_and_not_text 'oc get scc/privileged -o yaml' "system:serviceaccount:$(oc project -q):fake-sa"
+os::cmd::expect_success 'oadm policy remove-scc-from-group privileged fake-group'
+# os::cmd::expect_failure 'oadm policy add-scc-to-group privileged fake-group' # this doesn't fail for some reason
```

Comment:
For reference, these [two lines](https://github.com/stevekuznetsov/origin/blob/master/test/cmd/admin.sh#L100-L101) originally were:
```
oadm policy remove-scc-from-group privileged fake-group
[ ! "$(oadm policy add-scc-to-group privileged fake-group)" ]
```